### PR TITLE
Add simple serde importer

### PR DESCRIPTION
**Pull request by Areredify**
Right now you need to duplicate your uuids when defining a trivially (de)serializable struct, I'll add a macro to unify this later.

### CODE REVIEW

**Comment by Areredify** [comment]
Ron syntax is a little awkward with this uuid serialing variant but other variants (from 3 total) are either not working or even more awkward. You can look at examples in tests.

**Comment by kabergstrom** [comment]
I think it looks good!

Just two points of feedback:
- The RON importer should be behind a feature flag that is enabled by default
- As you mentioned, ideally the system would use the UUID from the `#[uuid = "..."]` attribute

**Comment by Areredify** [comment]
No, typetag doesn't support multiple structures in one map, at least judging by the documentation. I've looked into the implementation too and it doesn't seem to be possible either without changing the implementation.

**Comment by kabergstrom** [comment]
@Areredify Do you want me to merge this now and we'll try to figure out a better derive later on?

**Comment by Areredify** [comment]
No, I'll add better derive today

File: Cargo.lock [hunk 1]
```diff
@@ -541,7 +541,9 @@ dependencies = [
  "inventory 0.1.3 (registry+https://github.com/rust-lang/crates.io-index)",
  "ron 0.4.2 (registry+https://github.com/rust-lang/crates.io-index)",
  "serde 1.0.97 (registry+https://github.com/rust-lang/crates.io-index)",
+ "serde-importable-derive 0.1.0",
  "type-uuid 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
+ "typetag 0.1.3 (registry+https://github.com/rust-lang/crates.io-index)",
  "uuid 0.7.4 (registry+https://github.com/rust-lang/crates.io-index)",
```
Comment (jojolepro) [inline Cargo.lock]: Why is the Cargo.lock file in the git repo?

Comment (Areredify) [reply Cargo.lock]: No idea, do you want me to add it to gitigbore?

Comment (jojolepro) [reply Cargo.lock]: I never worked in this repo, but usually the .lock files are in the gitignore ^^

Comment (Moxinilian) [reply Cargo.lock]: I believe the new guidelines are to leave it on the repo. Why it is like this however is unknown to me.

Comment (kabergstrom) [reply Cargo.lock]: Yeah the guidelines do state so for binaries: https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries

But atelier-assets is not primarily something that should be distributed independently, so I think it's fine to add it to the gitignore @Areredify !

**Comment by Areredify** [comment]
Yeah, I'll remove `cargo.lock`s with a separate pr

**Pull request merged**